### PR TITLE
Removed  statements exposing the tokens on client.

### LINKED ISSUE #91: Console logs exposing tokens on client.
## Issue(s)
There quite a few console logs exposing tokens from the Client on Production.

## ToDo
Remove all the console logs from the Client.

### PR DESCRIPTION
**Pull request by sourabhkumar45**
Removed console.log statements from navbar.js and  App.js

### CODE REVIEW

**Review by apurv-1 — CHANGES_REQUESTED** [review]
Hello @saurabh-33, This is good! But there are still few console logs on game.js and lobby.js.
It would be great if you can remove those?
And for that, you don't have to close this PR.
Just remove those console logs and commit changes on the same branch.

**Comment by sourabhkumar45** [comment]
Do I have remove all the console.logs or only those that are displaying data of currentUser.

**Comment by apurv-1** [comment]
> Do I have remove all the console.logs or only those that are displaying data of currentUser.

Yes. Only ignore the file serviceWorker.js

File: client/src/components/login/login.js [hunk 1]
```diff
@@ -85,7 +85,6 @@ class Login extends Component {
   }
 
   handleGoogleOAuthError(response) {
```
Comment (apurv-1) [inline client/src/components/login/login.js:87]: This was a error message brother! @fave77 we can have toast message for this?

**Review by apurv-1 — APPROVED** [review]

File: client/src/components/login/login.js (hunk 1 above)
Comment (fave77) [reply client/src/components/login/login.js:87]: @apurv-1 sure

**Review by fave77 — APPROVED** [review]

**Pull request merged**